api/src/main/java/org/pmiops/workbench/institution/InstitutionUserInstructionsMapper.java

```diff
@@ -1,37 +1,36 @@
 package org.pmiops.workbench.institution;
 
-import org.mapstruct.AfterMapping;
-import org.mapstruct.Context;
+import com.google.common.base.Strings;
 import org.mapstruct.Mapper;
-import org.mapstruct.Mapping;
-import org.mapstruct.MappingTarget;
 import org.owasp.html.HtmlPolicyBuilder;
 import org.owasp.html.PolicyFactory;
 import org.pmiops.workbench.db.model.DbInstitution;
 import org.pmiops.workbench.db.model.DbInstitutionUserInstructions;
+import org.pmiops.workbench.exceptions.BadRequestException;
 import org.pmiops.workbench.model.InstitutionUserInstructions;
 import org.pmiops.workbench.utils.mappers.MapStructConfig;
 
 @Mapper(config = MapStructConfig.class)
 public interface InstitutionUserInstructionsMapper {
-  @Mapping(target = "institutionUserInstructionsId", ignore = true)
-  @Mapping(target = "institution", ignore = true) // set by setFields()
-  @Mapping(target = "userInstructions", ignore = true) // set by setFields()
-  DbInstitutionUserInstructions modelToDb(
-      InstitutionUserInstructions modelObject, @Context InstitutionService institutionService);
+  default DbInstitutionUserInstructions modelToDb(
+      InstitutionUserInstructions modelObject, InstitutionService institutionService) {
```

Comment:
style-wise: I would just pass the short name in as the second parameter instead of the whole service. It's a "law of demeter" thing: You hand the ATM your debit card, not your jacket with the pocket with the wallet with the card in it.  As a bonus, this would allow other services to use the mapper more easily.

Also, do you not need `@Context` here?